app/Models/BookingRequest.php

```diff
@@ -99,4 +99,12 @@ public function reviewers()
     public function scopePending(Builder $query) {
         $query->where('status', 'review');
     }
+
+    /**
+     * Get the rooms that are part of the booking request.
+     */
```

Comment:
have you been hit by the copy pasta bandit?

Reply:
@danseremet ^

Reply:
fixed